src/backend/opengl/CommandBufferGL.cpp

```diff
@@ -236,11 +253,11 @@ namespace opengl {
                     {
                         DrawArraysCmd* draw = commands.NextCommand<DrawArraysCmd>();
                         if (draw->firstInstance > 0) {
-                            glDrawArraysInstancedBaseInstance(GL_TRIANGLES,
+                            glDrawArraysInstancedBaseInstance(GLPrimitiveTopology(lastRenderPipeline->GetPrimitiveTopology()),
```

Comment:
The translated primitive topology could be store in the ```opengl::RenderPipeline``` to avoid work at command buffer submit time.